index.html

```diff
@@ -104,9 +108,216 @@ <h2 class="color-heading" id="heading-2">
+                <li>HTML</li>
+              </ul>
+            </li>
+            <li class="display-flex-width btn-project-1">
+              <button class="action-button" type="button">See Project</button>
+            </li>
+          </ul>
+        </div>
+
+        <div class="section-container3 section-box item-grid">
+          <ul class="first-work display-flex-3">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project3.png"
+                alt="Health and Fitness Project: Personalized Stories selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">Health and Fitness Project</h3>
+            </li>
+            <li class="display-flex-width languages-used">
+              <ul class="languages language-flex">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+            </li>
+            <li class="display-flex-width btn-project-1">
+              <button class="action-button" type="button">See Project</button>
+            </li>
+          </ul>
+        </div>
+        <div class="section-container2 section-box item-grid">
+          <ul class="first-work display-flex-4">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project4.png"
+                alt="Art Prinitngs: Personalized Art Selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">Art Printing Portfolio</h3>
+            </li>
+            <li class="display-flex-width languages-used">
+              <ul class="languages language-flex">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+            </li>
+            <li class="display-flex-width btn-project-1">
+              <button class="action-button" type="button">See Project</button>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-me">
+      <div class="flex-about">
+        <div class="about-heading"><h3>About Me</h3></div>
+        <div class="about-para">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="about-button">
+          <button class="about-btn" type="button">Get My Resume</button>
+        </div>
```

Comment:
- [ ] The button (e.g., "Get my resume") is a link a or button tag, and follow the button styling: Active, Hover and Pressed, Kindly work on Hover and Pressed.

Reply:
``` 
I just need to add these effects to "GET RESUME" button
``` (I have added my button effects to "My Recent Works")

.action-button:hover {
        cursor: pointer;
        background: white;
        color: green;
  }
```